lang/main.json

```diff
@@ -72,6 +72,8 @@
     "toolbar": {
         "addPeople": "Add people to your call",
         "audioonly": "Enable / Disable audio only mode (saves bandwidth)",
+        "feedback": "Leave feedback",
```

Comment:
The toolbar bundle is not alphabetically ordered, I see however that it's not only your change that made it end up like this. Do you think it would be worth the cost here to get it ordered?

Reply:
I just got confirmation from yana that there is nothing stopping me from doing so. How about I do it in a follow up commit? I'll make a ticket for myself to do so if you're okay with that.

Reply:
Nevermind, I think it's better if I alpha-order this section now but do the rest of the file in a follow up commit.